src/udev/udev-worker.c

```diff
@@ -248,7 +248,7 @@ void udev_broadcast_result(sd_device_monitor *monitor, sd_device *dev, EventResu
                                 (void) device_add_property(dev, "UDEV_WORKER_ERRNO_NAME", str);
                         break;
                 }
-                case EVENT_RESULT_EXIT_STATUS_BASE ... EVENT_RESULT_EXIT_STATUS_MAX:
+                case (EVENT_RESULT_EXIT_STATUS_BASE+1) ... EVENT_RESULT_EXIT_STATUS_MAX:
                         (void) device_add_propertyf(dev, "UDEV_WORKER_EXIT_STATUS", "%i", result - EVENT_RESULT_EXIT_STATUS_BASE);
                         break;
 
```

Comment:
maybe on the next case statement also add `case EVENT_RESULT_SUCCESS`, so that it is covered by the same `assert_not_reached()` statement, since that should of course never happen.